packages/lodestar/src/chain/stateTransition/util/attestation.ts

```diff
@@ -104,6 +105,27 @@ export function isValidIndexedAttestation(
   return true;
 }
 
+export function verifyAttestation(
+  config: IBeaconConfig,
+  state: BeaconState,
+  attestation: Attestation): boolean {
+  const validatorIndexes = getAttestingIndices(config, state, attestation.data, attestation.aggregationBits);
+  const attesters = validatorIndexes.map(index => state.validators[index].pubkey);
```

Comment:
This would mean that every validator approved(signed this attestation). You should calculate CrosslinkComittee and it will return sequence of validator indexes that are suppose to sign this attestation. so for example:
```
comittee 4, 19, 22, 45
bits:    0,  1,  1,  0
```
means that validators with index 19 and 22 signed attestation and only their pubkeys should go in verifyMultiple

Reply:
But, let's put this on hold a bit until we figure out what other things should be checked as well

Reply:
not related to this PR, what I see is our `processAttestation` is not the same to the spec.